website/docs/docs/deploy/jobs.md

```diff
@@ -0,0 +1,29 @@
+---
+title: "Jobs in dbt Cloud"
+sidebar_label: "Jobs"
+description: "Learn about deploy jobs and continuous integration (CI) jobs in dbt Cloud and what their differences are." 
+tags: [scheduler]
+---
+
+In dbt Cloud, you can create and set up triggers for these jobs:
+- [Deploy jobs](/docs/deploy/deploy-jobs)
+- [Continuous integration (CI) jobs](/docs/deploy/continuous-integration)  
+
+:::tip Join our beta 
```

Comment:
should this be a reusable? Just noticed it's slightly different than the [one above](https://github.com/dbt-labs/docs.getdbt.com/pull/3899/files#diff-b5cfa135eb077190c1e80ef584693ca77eb4c5cb8b0600390e1d66bd1764bb6eR20). But maybe that's intentional?

Reply:
i thought about that too. but each one is slightly different than the other. the one here needs to include both since the page talks about diff types of dbt cloud jobs (deploy jobs, ci jobs)